src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,57 @@
+package study;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+
+  private Set<Integer> numbers;
+
+  @BeforeEach
+  void setUp() {
+    numbers = new HashSet<>();
+    numbers.add(1);
+    numbers.add(1);
+    numbers.add(2);
+    numbers.add(3);
+  }
+
+  @DisplayName("사이즈 확인")
+  @Test
+  void getSizeTest() {
+    assertThat(numbers).size().isEqualTo(3);
```

Comment:
```suggestion
        assertThat(numbers).hasSize(3);
```

collection의 경우 위와 같이 구현도 가능함